oauth2_http/java/com/google/auth/oauth2/ServiceAccountJwtAccessCredentials.java

```diff
@@ -297,14 +296,17 @@ public static ServiceAccountJwtAccessCredentials fromStream(
     if (fileType == null) {
       throw new IOException("Error reading credentials from stream, 'type' field not specified.");
     }
-    if (SERVICE_ACCOUNT_FILE_TYPE.equals(fileType)) {
+    if (GoogleCredentials.GoogleCredentialsInfo.SERVICE_ACCOUNT_CREDENTIALS
+        .getFileType()
+        .equals(fileType)) {
       return fromJson(fileContents, defaultAudience);
     }
     throw new IOException(
         String.format(
             "Error reading credentials from stream, 'type' value '%s' not recognized."
                 + " Expecting '%s'.",
-            fileType, SERVICE_ACCOUNT_FILE_TYPE));
+            fileType,
+            GoogleCredentials.GoogleCredentialsInfo.SERVICE_ACCOUNT_CREDENTIALS.getFileType()));
```

Comment:
nit: These usages are a bit verbose. perhaps do a static import for the specific enum val?

Reply:
Updated to be less verbose and dropped the `GoogleCredentials.GoogleCredentialsInfo`